media/js/firefox/whatsnew/whatsnew-66.js

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+    (function() {
+        'use strict';
+
+        var buttons = document.querySelectorAll('.wn66-benefit-link');
+        var content = document.getElementById('wn66-modal');
+        var client = Mozilla.Client;
+
+        for (var i = 0, len = buttons.length; i < len; i++) {
+            var button = buttons[i];
+            button.addEventListener('click', openModal);
+            button.setAttribute('aria-role', 'button');
+        }
+
+        function openModal(e) {
+            e.preventDefault();
+
+            var trigger = e.target;
+            var title = trigger.textContent;
+            var app = 'mzp-t-firefox app-' + trigger.dataset.app;
+
+            Mzp.Modal.createModal(e.target, content, {
+                title: title,
+                className: app,
+                closeText: window.Mozilla.Utils.trans('global-close')
+            });
+        }
+
+        function checkUpToDate() {
+            // bug 1419573 - only show "Your Firefox is up to date" if it's the latest version.
+            if (client.isFirefoxDesktop) {
+                client.getFirefoxDetails(function(data) {
+                    if (data.isUpToDate) {
+                        document.querySelector('.c-page-header').classList.add('is-up-to-date');
+                    }
+                });
+            }
+        }
+
+        /*
+        * Sticky CTA & bottom CTA
+        */
+
+        var $stickyCTA = $(document.querySelectorAll('.c-sticky-signup'));
```

Comment:
Not a blocker, but any reason to switch to jQuery here? (we have hopes and dreams of removing it from bedrock one day, so the less we write the better 🙂).